Author: Dmitry Smirnov <onlyjob@member.fsf.org>
From: Thomas Richter <richter@rus.uni-stuttgart.de>
Subject: Cannot disable smooth scrolling
Origin: other, http://bugs.debian.org/624095
Bug-Debian: http://bugs.debian.org/624095
Last-Update: 2011-11-21
Forwarded: yes
Description: Disable smooth scrolling
 Commenting code which is enabling smooth scrolling disregarding preferences
 to leave decision whenever use the smooth scrolling or not to user.

--- a/src/text/fmt/xp/fv_View.cpp
+++ b/src/text/fmt/xp/fv_View.cpp
@@ -14339,16 +14339,19 @@
 	if(static_cast<UT_sint32>(m_iNumHorizPages) > m_pLayout->countPages())
 	{
 		m_iNumHorizPages = m_pLayout->countPages();
 	}
+/*	Debian bug #624095: smooth scrolling enabled, preferences ignored.
+ *	Commenting to leave decision whenever use the smooth scrolling to user.
 	if (m_iNumHorizPages > 1)
 	{
 		XAP_App::getApp()->setEnableSmoothScrolling(false);
 	}
 	else
 	{
 		XAP_App::getApp()->setEnableSmoothScrolling(true);
 	}
+*/
 	if(iOldNo != static_cast<UT_sint32>(m_iNumHorizPages))
 	{
 		UT_uint32 iPrevYOffset = m_yScrollOffset;
 		XAP_Frame * pFrame = static_cast<XAP_Frame*>(getParentData());
